tests/conftest.py

```diff
@@ -164,6 +164,6 @@ def schema():
             },
         }, {
             '$ref': 'http://inveniosoftware.org/schemas/'
-                    'oaiserver/internal-v1.0.0.json',
+                    'oaiserver/internal-v1.1.0.json',
```

Comment:
I had to change the schema version here, otherwise the test_percolator was failing (as I removed the `[_oai][updated]` property). Is it ok? Do we need separate tests for `internal-v1.0.0.json` schema?